src/openrct2/actions/RideSetStatus.hpp

```diff
@@ -25,6 +25,13 @@
 #include "../world/park.h"
 #include "../ride/ride.h"
 
+static rct_string_id _StatusErrorTitles[] =
+{
+    STR_CANT_CLOSE,
+    STR_CANT_OPEN,
+    STR_CANT_TEST
```

Comment:
Wouldn't Close-test-open be a bit easier?
(No biggie, just wondering.)

Reply:
It's in order of ride_status so i can't change the order.